drivers/hsc/include/hsc_internals.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (C) 2020 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_hsc
+ * @brief       Internal addresses, constants for the HSC sensor.
+ * @{
+ *
+ * @file
+ * @brief       Internal addresses, constants for the HSC sensor.
+ *
+ * @author      Quang Pham <phhr_quang@live.com>
+ */
+
+#ifndef HSC_INTERNALS_H
+#define HSC_INTERNALS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    HSC I2C Address
+ * @{
+ */
+#define HSC_ADDR                    (0x28)   /* 7 bit address */
```

Comment:
```suggestion
#ifndef HSC_PARAM_ADDR
#define HSC_PARAM_ADDR              (0x28)   /* 7 bit address */
#endif
```

To follow the naming convention and allow to overwrite the address as sensors can have different addresses.